README.md

```diff
@@ -57,20 +58,25 @@ provider "ibm" {
   region           = "us-south"
 }
 module "schematics_agent" {
-  source                    = "terraform-ibm-modules/schematics-agent/ibm"
-  version                   = "latest" # Replace "latest" with a release version to lock into a specific release
-  infra_type                = "ibm_kubernetes" # ibm_kubernetes, ibm_openshift, ibm_satellite.
-  cluster_id                = "<cluster-id>"
-  cluster_resource_group_id = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
-  cos_instance_name         = "<cos-instance-name>"
-  cos_bucket_name           = "<cos-bucket-name>"
-  cos_bucket_region         = "<cos-bucket-region>"
-  agent_location            = "us-south"
-  agent_description         = "schematics agent description"
-  agent_name                = "k8s-schematics-agent"
-  agent_resource_group_id   = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
-  schematics_location       = "us-south" # Allowed values are `us-south`, `us-east`, `eu-gb`, `eu-de`.
-  agent_version             = "<agent-version>"
+  source                      = "terraform-ibm-modules/schematics-agent/ibm"
+  version                     = "latest" # Replace "latest" with a release version to lock into a specific release
+  infra_type                  = "ibm_kubernetes" # ibm_kubernetes, ibm_openshift, ibm_satellite.
+  cluster_id                  = "<cluster-id>"
+  cluster_resource_group_name = "Default"
+  cos_instance_name           = "<cos-instance-name>"
+  cos_bucket_name             = "<cos-bucket-name>"
+  cos_bucket_region           = "<cos-bucket-region>"
+  agent_location              = "us-south"
+  agent_description           = "schematics agent description"
+  agent_name                  = "k8s-schematics-agent"
+  agent_resource_group_name   = "Default"
+  schematics_location         = "us-south" # Allowed values are `us-south`, `us-east`, `eu-gb`, `eu-de`.
+  agent_version               = "<agent-version>"
+  agent_tags                  = []
+  agent_metadata_name         = <agent-metadata-name>
+  agent_metadata_value        = <agent-metadata-value>
+  run_destroy_resources       = <run-destroy-resources>
+  agent_state                 = <agent-state>
```

Comment:
please use real sample mock values for these inputs so users get an idea of the usage and format required. You also don't need to include every input - just the required ones. All inputs are listed in the table by the generated by the hook